en/concept/events_interface.md

```diff
@@ -0,0 +1,133 @@
+# Events Interface
+
+The *Events Interface* provides a system-wide API to inform a user about certain events.
+It is meant to replace existing usage of `mavlink_log_*` calls.
+
+## Usage
+### Basic
+To use the API, add this include:
+```cpp
+#include <px4_platform_common/events.h>
+```
+And then define and send the event from the desired code location:
+```cpp
+events::send(events::ID("mymodule_test"), events::Log::Info, "Test Message");
+```
+
+#### Backward compatibility
+
+For older GCS versions without events interface support, PX4 currently sends out all events also as `mavlink_log_*` `STATUSTEXT` message.
+In addition, the message must be tagged with an appended tab (`\t`) so that newer GCS's can ignore that and only show the event.
+
+So whenever adding an event, be sure to also add a `mavlink_log_` call. For example:
+```cpp
+mavlink_log_info(mavlink_log_pub, "Test Message\t");
+events::send(events::ID("mymodule_test"), events::Log::Info, "Test Message");
+```
+All such `mavlink_log_` calls will be removed after the next release.
```

Comment:
So we will have to remove this section in next release?

Reply:
Correct, after the next release.